packages/nuekit/test/nuekit.test.js

```diff
@@ -121,7 +121,7 @@ test('get data', async () => {
 
 test('content collection', async () => {
   // Default sorting is on pubDate returning most recent first.
-  await write('blog/first-a.md', '# First')
+  await write('blog/first-a.md', createFront('First'))
   await write('blog/first-b.md', createFront('Second', '2020-01-04'))
```

Comment:
```suggestion
  await write('blog/first-b.md', createFront('Second', '2020-01-04'))
  await write('blog/first-c.md', '# Third')
```

Additionally, to the default fallback of that front-matter function (`2020-01-20`, see [line 49](https://github.com/nuejs/nue/pull/190/files#diff-c10aeaf246848a1abdde8aecf8d19aefb90b27781ed29d8f96e1957a1cb9dc96R49)), I would also add a test case where the `pubDate` field is `undefined`.

Also see https://github.com/nuejs/nue/commit/da78e4f71fb9c2d28ee7fb72765c6f98401ba37e#commitcomment-138287265

What do you think?

Reply:
Nevermind. It looks like you want to do that or something similar in #191

Reply:
Agreed. Yes I think I'very completed the test-cases in #191

Reply:
One more reason I did not add such test-case to this PR was because that
Currently the order of arr returned by `site.getContentCollection` can vary and is not predictable when pubDate is not provided via `createFront`. See https://github.com/fritx/nue/actions/runs/7815178425/job/21317944828

If we want predictable results we have to fix/complete the sorting logic.
If we don't need predictable results we can not add such test-cases as they can fail in CI...
cc @nobkd